### PR TITLE
Tfiac component documentation

### PR DESCRIPTION
**Pull request by mellado**
**Description:**
This is a PR to update the documentation of a new component that already has a PR in the main home-assistant repo.
New componenet pull request URL:
https://github.com/home-assistant/home-assistant/pull/21823

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/climate.tfiac.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+date: 2019-03-10 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tfiac.png
+ha_category: Climate
+ha_release: 0.88
+ha_iot_class: "Local Polling"
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/climate.tfiac.markdown]: Please remove this

File: source/_components/climate.tfiac.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+title: "Tfiac AC"
+description: "Instructions on how to integrate Tfiac AC devices with Home Assistant."
+date: 2019-03-10 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tfiac.png
+ha_category: Climate
+ha_release: 0.88
```
Comment (klaasnicolaas) [inline source/_components/climate.tfiac.markdown]:
```suggestion
ha_release: "0.90"
```

File: source/_components/climate.tfiac.markdown [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## {% linkable_title Configuration %}
+
+To add your Tfiac device into your Home Assistant installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Full manual example configuration.yaml entry
+climate:
+  - platform: tfiac
+    host: 192.168.1.108
```
Comment (klaasnicolaas) [inline source/_components/climate.tfiac.markdown]:
```suggestion
    host: IP_ADDRESS
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/climate.tfiac.markdown (hunk 1 above)
Comment (mellado) [reply source/_components/climate.tfiac.markdown]: Not sure if you are suggesting to delete lines 13 and 14.

Comment (klaasnicolaas) [reply source/_components/climate.tfiac.markdown]: Line 14 and 15

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by mellado** [comment]
Thanks!!!
My first PR ever!!!
Do I have to close the PR?

**Comment by fabaff** [comment]
> Do I have to close the PR?

No, it will be merge after #21823 is included.

**Comment by klaasnicolaas** [comment]
✅ When the deploy is ready it can be merged.

**Review by klaasnicolaas — APPROVED** [review]

**Comment by tammeryousef1006** [comment]
its not working the integration , what to do
Setup of platform tfiac is taking longer than 60 seconds. Startup will proceed without waiting any longer

**Comment by frenck** [comment]
Hi, there @tammeryousef1006.

Please don't use old/handled PRs to ask for help or report issues.

If you need help or want to report an issue: check our website on where to do that:

<https://www.home-assistant.io/help>

Thanks! 👍 

../Frenck 🚂

**Comment by tammeryousef1006** [comment]
ok boss

**Pull request merged**